pull/github.go

```diff
@@ -113,6 +114,7 @@ type GitHubContext struct {
 	owner  string
 	repo   string
 	number int
+	draft  *bool
```

Comment:
Can you read this as a property of `pr` rather than making a new top-level field?